library/src/main/scala/giter8/TemplateRenderer.scala

```diff
@@ -21,14 +21,14 @@ import java.io.File
 
 trait TemplateRenderer {
   def render(baseDirectory: File,
-             outputDirectory: File,
+             outputDirectory: Either[File, File],
```

Comment:
What does the `Either[File, File]` mean? Does `TemplateRenderer` have to know if it's Left or Right?